base/compiler/ssair/verify.jl

```diff
@@ -233,71 +242,76 @@ function verify_ir(ir::IRCode, print::Bool=true,
                 end
                 check_op(ir, domtree, val, Int(edge), last(ir.cfg.blocks[stmt.edges[i]].stmts)+1, idx, print, false, i, allow_frontend_forms)
             end
-        elseif isa(stmt, PhiCNode)
-            for i = 1:length(stmt.values)
-                val = stmt.values[i]
-                if !isa(val, SSAValue)
-                    @verify_error "Operand $i of PhiC node $idx must be an SSA Value."
-                    error("")
-                end
-                if !isa(ir[val][:inst], UpsilonNode)
-                    @verify_error "Operand $i of PhiC node $idx must reference an Upsilon node."
-                    error("")
-                end
-            end
-        elseif (isa(stmt, GotoNode) || isa(stmt, GotoIfNot) || isexpr(stmt, :enter)) && idx != last(ir.cfg.blocks[bb].stmts)
-            @verify_error "Terminator $idx in bb $bb is not the last statement in the block"
-            error("")
+        elseif stmt === nothing
+            # Nothing to do
         else
-            if isa(stmt, Expr) || isa(stmt, ReturnNode) # TODO: make sure everything has line info
-                if (stmt isa ReturnNode)
-                    if isdefined(stmt, :val)
-                        # TODO: Disallow unreachable returns?
-                        # bb_unreachable(domtree, Int64(edge))
-                    else
-                        #@verify_error "Missing line number information for statement $idx of $ir"
+            is_phinode_block = false
```

Comment:
Just a cosmetic suggestion: How about using `continue` in the `isa(stmt, PhiNode)` and `stmt === nothing` branches? Then we can remove one level indentation for the last `else` clause.

Reply:
I agree that's better